recipes/range-v3/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.12.0" %}
+
+package:
+  name: range-v3
+  version: {{ version }}
+
+source:
+  url: https://github.com/ericniebler/range-v3/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 015adb2300a98edfceaf0725beec3337f542af4915cec4d0b89fa0886f4ba9cb
+
+build:
+  number: 0
+  run_exports:
+    # Let's keep this conservative.
+    - {{ pin_subpackage('range-v3', max_pin='x.x') }}
```

Comment:
range-v3 is a header only library - applications built against it should not need it installed at runtime.